src/core/mdn-annotation.js

```diff
@@ -188,3 +198,29 @@ function findElements(data) {
   const elemsWithId = document.body.querySelectorAll("[id]:not(script)");
   return [...elemsWithId].filter(({ id }) => Array.isArray(data[id]));
 }
+
+/**
+ * @param {MdnEntry['engines']} engines
+ * @returns {HTMLSpanElement|undefined}
+ */
+function getEngineSupportIcons(engines) {
+  if (engines.length === 3) {
+    return html`<span title="${l10n.inAllEngines}">✅</span>`;
+  }
+  if (engines.length < 2) {
+    return html`<span title="${l10n.inSomeEngines}">🚫</span>`;
+  }
+}
+
+/**
+ * @param {MdnEntry['engines']} engines
+ * @returns {HTMLParagraphElement|undefined}
+ */
+function getEngineSupport(engines) {
+  if (engines.length === 3) {
```

Comment:
Why 3 when `MDN_BROWSERS` includes many browsers?

Reply:
"blink", "gecko" and "webkit"

Reply:
Ah okay, it's from https://github.com/w3c/mdn-spec-links/blob/346c5fac134f99f4bd52920a1be1542f661aede6/.browser-compat-data-process.js#L365.